### PR TITLE
chore(docs): Update snippet 'global-state-vs-local-state'

### PR DESCRIPTION
**Pull request by Phhansen**
Add code examples

### CODE REVIEW

**Comment by Karnaukhov-kh** [comment]
Hi. 

Looks good :). There is a request for "RxState as a global state service" snippet (#113 & #111). "Moving the tasks array to our global state" part covers this case really well as for me. So maybe it can be a separate snippet?

**Comment by Phhansen** [comment]
@Karnaukhov-kh that is a good idea! I'll look into it later tonight 👍

**Comment by BioPhoton** [comment]
Hi @Phhansen 

What's the status on that? Is this finished and discussed stuff is somewhere else?

Just want to get an idea of how we deal with the feedback.

**Comment by Phhansen** [comment]
Hi @BioPhoton, 

Sorry for the delay! I've been away on holiday with my family and just got back this weekend. I'll be at my computer tonight and able to finish this up.

There has been no discussion anywhere else apart from this PR.

Should I make another snippet and include a few ways of doing a global service? Using `injection token`, `service` with `providedIn: "root"` and such?

**Comment by BioPhoton** [comment]
@Karnaukhov-kh and I discussed that and somewhere put our decision regarding whst the best way for a global service is. I'll try if I can point you there later.

**Comment by Phhansen** [comment]
And i´m back from holidays! Any chance you´ve found your discussion, and what actions am I to take from now?

@BioPhoton

**Comment by BioPhoton** [comment]
@Phhansen I guess we can also adopt the way how to use it globally. 
We will just adopt the PR. You can start.

**Comment by Phhansen** [comment]
@BioPhoton I'm still a little unsure what you want me to start on. You mentioned some notes, but I haven't seen anything yet. I'll get right to it once I'm sure what it is :D

File: libs/state/docs/snippets/global-state-vs-local-state.md [hunk 1]
```diff
@@ -31,18 +31,172 @@ Let's take a look at a simple todo app. This app has 2 views.
...
+```typescript
+import { GLOBAL_RX_STATE } from './rx-state';
+
+@Component({
+  selector: 'my-app',
+  templateUrl: './app.component.html',
+  styleUrls: [ './app.component.css' ]
+})
+export class AppComponent  {
+constructor(@Inject(GLOBAL_RX_STATE) private state, private tasksService: TasksService) {
```
Comment (BioPhoton) [inline libs/state/docs/snippets/global-state-vs-local-state.md:143]: How are those 2 services related to ech other? Also the state is not used..

**Comment by BioPhoton** [comment]
Hi @Phhansen some related issue is here #113 .

The token approach is ok. 

If you just could move the code in a separate file. So we have it separate form this topic. 

Thanks! :)

File: libs/state/docs/snippets/global-state-vs-local-state.md (hunk 1 above)
Comment (Phhansen) [reply libs/state/docs/snippets/global-state-vs-local-state.md:143]: Inside the `constructor` there is the `connect` to the tasksService.

```
    /* Fetch tasks from backend */
    this.connect("tasks", this.tasksService.fetchTasks());
```

I think my original thought was that if two components are using the same state, they shouldn´t both call the API, so the parent should. At least from my memory.

If anything needs changing I'll do just that.

**Comment by BioPhoton** [comment]
Nice! Thanks for the contribution!! <3

**Pull request merged**